.github/workflows/panvimdoc.yml

```diff
@@ -20,6 +21,7 @@ jobs:
           vimdoc: hawtkeys
           version: "NVIM >=0.9"
           toc: true
+          description: "For NVIM >=0.9"
```

Comment:
Shouldn't this already be added automatically?

Reply:
yeah, but i don't want the date portion as it causes unnecessary commits

Reply:
ah, gotcha - lgtm then